monitoring/generator.go

```diff
@@ -104,6 +104,25 @@ func (r Row) validate() error {
 	return nil
 }
 
+// ObservableOwner denotes a team that owns an Observable. The current teams are described in
+// the handbook: https://about.sourcegraph.com/handbook/engineering/2021_org
+type ObservableOwner string
+
+const (
+	// Core Products teams
```

Comment:
```suggestion
	// Core products teams
```

Unless you copied from handhook?

Reply:
it's from [this page](https://about.sourcegraph.com/handbook/engineering/2021_org)

<img width="362" alt="image" src="https://user-images.githubusercontent.com/23356519/88031644-6f757b00-cb6f-11ea-92d4-a80f64dc79dd.png">

Reply:
I like the lower case though